### ISSUE TITLE
AttributeError: 'RKI' object has no attribute 'get_new_confirmed' & 'get_confirmed'

### ISSUE THREAD

**Issue by paulogonc** [Priesemann-Group/covid19_inference#45]
Hi,

there is a small bug in the jupyter notebook  "example_one_bundesland.ipynb" at the create model section. I guess the call needs to be changed as follows:

```
total_cases = rki.get_total(value="confirmed", bundesland="Sachsen",data_begin=bd, data_end=ed)
new_cases_obs = rki.get_new(value="confirmed", bundesland="Sachsen",data_begin=bd, data_end=ed)```

Best regards
Paulo

**Comment by semohr** [comment]
The `example_one_bundesland.ipynb` wasn't updated for some time but this problem will be fixed in the next main version.
Was already fixed on commit https://github.com/Priesemann-Group/covid19_inference/commit/e285ca5636e6b02a0ebdbc6177f9723944c7e2fa, the branch just has to be merged.

Nevertheless, thanks a lot for your input and help.

**Comment by semohr** [comment]
Merged to master b39a992ec974c1952aaab8175660c3031e4e8824
